.github/workflows/test-and-release.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
       APP_REG_CLIENT_ID: ${{ secrets.APP_REG_CLIENT_ID }}
       B2C_CLIENT_SECRET: ${{ secrets.B2C_CLIENT_SECRET }}
       B2C_TENANT_ID: ${{ secrets.B2C_TENANT_ID }}
+      B2C_USER_FLOW_NAME: B2C_1_developer
```

Comment:
I assume this change is needed for CI to work.

Reply:
Previously, it is hardcode. So, making it as parameter will give us more flexible to test somewhere else

Reply:
This change is needed for CI to work